com.unity.netcode.gameobjects/Runtime/NetworkVariable/NetworkVariableBase.cs

```diff
@@ -13,14 +13,11 @@ public void Initialize(NetworkBehaviour networkBehaviour)
         {
             m_NetworkBehaviour = networkBehaviour;
         }
-        /// <summary>
-        /// The name of the channel to be used for syncing
-        /// </summary>
-        public const NetworkChannel NetworkVariableChannel = NetworkChannel.NetworkVariable;
 
-        protected NetworkVariableBase(NetworkVariableReadPermission readPermIn = NetworkVariableReadPermission.Everyone)
+        protected NetworkVariableBase(NetworkVariableReadPermission readPermIn = NetworkVariableReadPermission.Everyone, NetworkDelivery networkDelivery = NetworkDelivery.ReliableSequenced)
```

Comment:
Here's the thing, when we go to snapshotting users won't have a per-variable choice anymore; this is why I was going to make this a static property of the NetworkVariable (and ClientNetworkVariable)

Reply:
@SamuelBellomo here too

Reply:
based on our Slack conversation, reverted my change in this PR and will defer to later conversations